harness/python/feature.py

```diff
@@ -127,3 +124,20 @@ async def check_result(self, handle: WorkflowHandle) -> None:
                     raise TypeError("Unexpected activity error") from err
             else:
                 raise err
+
+    def start_worker(self):
+        """Creates and starts worker with the task queue & workflows/ activities set, if it is not
+        already running"""
+        if self.worker is None:
+            self.worker = Worker(
+                self.client,
+                task_queue=self.task_queue,
+                workflows=self.feature.workflows,
+                activities=self.feature.activities,
+            )
+            self.worker._start()
```

Comment:
If using public API, making this function `async def start_worker` and changing this line to `asyncio.create_task(self.worker.run())` would be the way to only use public API. No biggie currently though.